lib/browser/api/auto-updater/squirrel-update-win.js

```diff
@@ -113,3 +113,11 @@ exports.supported = function () {
     return false
   }
 }
+
+exports.getNugetPackageID = function () {
+  return path.basename(path.resolve(process.execPath, '..', '..'))
+}
+
+exports.getExeName = function () {
+  return path.basename(process.execPath).replace(/\.exe$/g, '')
```

Comment:
I think you can pass the extension to remove as a second argument to `path.basename`:

```js
return path.basename(process.execPath, '.exe')
```